### PR TITLE
Removing %LAUNCHER_ARGS% from arguments for portable apps

### PR DESCRIPTION
**Pull request by moozzyk**
Addresses #150

### CODE REVIEW

File: src/Microsoft.AspNetCore.Server.IISIntegration.Tools/WebConfigTransform.cs [hunk 1]
```diff
@@ -65,6 +65,12 @@ private static void TransformAspNetCore(XElement aspNetCoreElement, string appNa
             if (!isPortable)
             {
                 aspNetCoreElement.SetAttributeValue("processPath", appPath);
+                var arguments = (string)aspNetCoreElement.Attribute("arguments");
+                if (arguments != null && arguments.Contains("%LAUNCHER_ARGS%"))
+                {
```
Comment (Eilon) [inline src/Microsoft.AspNetCore.Server.IISIntegration.Tools/WebConfigTransform.cs]: Casing?

Comment (moozzyk) [inline src/Microsoft.AspNetCore.Server.IISIntegration.Tools/WebConfigTransform.cs]: will update

**Comment by moozzyk** [comment]
:up::date:

**Comment by Tratcher** [comment]
:shipit:

**Pull request merged**